tests/trainer_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail
+
+cd applications/trainer/upstream
+
+kustomize build base/crds | kubectl apply --server-side --force-conflicts -f -
+
+sleep 5
+kubectl wait --for condition=established crd/trainjobs.trainer.kubeflow.org --timeout=60s
+
+kustomize build overlays/manager | kubectl apply --server-side --force-conflicts -f -
+kubectl wait --for=condition=Available deployment/kubeflow-trainer-controller-manager -n kubeflow-system --timeout=180s
+
+kubectl apply --server-side --force-conflicts -f overlays/kubeflow-platform/kubeflow-trainer-roles.yaml
+kustomize build overlays/runtimes | kubectl apply --server-side --force-conflicts -f -
+
+kubectl get deployment -n kubeflow-system kubeflow-trainer-controller-manager
```

Comment:
please switch everything to kubeflow instead of kubeflow-system for now. We need an overlay for that.

Reply:
You can also go the other way around and add PSS to the ekubeflow-system namespace and add default blocker networkpolicies for kubeflow-system and exceptions for the trainer webhook trainer in kubeflow-system.